docs/reference/engine-target.md

```diff
@@ -208,7 +208,36 @@ The [BoundingBox](#class_engine.target.BoundingBox) class has the following publ
   - *score* is expected to be a number describing the prediction confidence.
 #### mot(with_confidence=True, frame=-1))
   Return the annotation in [MOT](https://motchallenge.net/instructions) format.
+#### coco()
+  Return the annotation in [COCO detection](https://cocodataset.org/#detection-2019) format.
+  For more information and a detailed description of COCO annotations, see this [COCO annotations tutorial](https://www.immersivelimit.com/tutorials/create-coco-annotations-from-scratch).
 
+### class engine.target.CocoBoundingBox
+Bases: `engine.target.BoundingBox`
+
+This target is used for 2D Object Detection and describes 2D bounding box in image plane containing an object of interest in [COCO detection](https://cocodataset.org/#detection-2019) format.
+A bounding box is described by the left-top corner and its width and height.
+The main difference with the `BoundingBox` target, is that the `CocoBoundingBox` target can also contain information on the object's segmentation.
+
+The [CocoBoundingBox](#class_engine.target.CocoBoundingBox) class has the following public methods:
+#### CocoBoundingBox(name, left, top, width, height, score=0, segmentation=[], iscrowd=0, area=0)
```

Comment:
note that the order of the arguments doesn't match the code

Reply:
I updated this.